livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -197,7 +192,7 @@ def __init__(
         *,
         instructions: str = "",
         modalities: list[api_proto.Modality] = ["text", "audio"],
-        model: OpenAIModel | str = DefaultOpenAIModel,
+        model: api_proto.OpenAIModel | str = api_proto.DefaultOpenAIModel,
```

Comment:
the other defaults are in this file as UPPER_CAMEL_CASE, rather than in api_proto